src/main/java/com/minecolonies/coremod/util/InventoryUtils.java

```diff
@@ -1769,19 +1769,4 @@ public static boolean removeStacksFromProvider(final ICapabilityProvider provide
 
         return true;
     }
-
-    /**
-     * Create a hard copy of the itemStack list also creating new ItemStack copies.
-     * @param list the list to be copied.
-     * @return the new list.
-     */
-    public static List<ItemStack> hardCopyListOfItemStacks(final List<ItemStack> list)
```

Comment:
Wasn't being used at all, I created it, its quite simple if we need it in the future we can create it again.